src/main/java/com/libertymutualgroup/herman/aws/lambda/LambdaBroker.java

```diff
@@ -169,6 +157,10 @@ public void brokerLambda() throws IOException {
         if (policy == null) {
             policy = defaultExecutionRole;
         }
+
+        String customAssumeRolePolicyFileName = Optional.ofNullable(this.configuration.getAssumeRolePolicy()).orElse("assume-role-policy.json");
+        String customAssumeRolePolicy = fileUtil.findFile(customAssumeRolePolicyFileName, true);
```

Comment:
It doesn’t look like this is used anywhere

Reply:
Yep, was a stray merge. Taken care of.